### PR TITLE
Russ/made in heights  pirouette

### PR DESCRIPTION
**Pull request by RussellSmith-Janrain**

### CODE REVIEW

**Comment by robotbill** [comment]
Please correct the formatting and change the link to a youtube link.

File: pirouette.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+*Spilling out onto counters, whirling like a pirouette*
+
+*Every salt, every shmear, everywhere I turn*
+
+*In my head, stitching bread until my bagels burn*
+
+*From the crease of your mouth to the picket fences*
+
+*All the bread, stretching out, blurring my defenses*
```
Comment (andrewsw-janrain) [inline pirouette.md]: minor nit that you don't need to fix:

    All the *dough*, stretching out, blurring my defenses

seems like an ideal bit

**Comment by andrewsw-janrain** [comment]
:heavy_check_mark:

**Comment by rkonell-janrain** [comment]
:+1:

**Comment by andrewsw-janrain** [comment]
I'm calling it.

**Comment by andrewsw-janrain** [comment]
thanks @RussellSmith-Janrain!

**Pull request merged**